app/assets/whitelist.txt

```diff
@@ -0,0 +1 @@
+tanayp@tamu.edu
```

Comment:
Maybe you can add all of our email address for everyone to testing the login, email address can be viewed on Heroku profile